pdf_form_fill/__openerp__.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2013 OpenERP s.a. (<http://openerp.com>).
+#    Copyright (C) 2013 initOS GmbH & Co. KG (<http://www.initos.com>).
+#    Author Nikolina Todorova <nikolina.todorova@initos.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Pdf form fill',
+    'version': '0.1.0',
+    'author': 'initOS GmbH & Co. KG',
+    'category': '',
+    'description': """
+The user should be able to fill .pdf form by mapping the fields from a record.
+The user should be able to download the filled .pdf form.
```

Comment:
As someone who admittedly is not familiar with PDF forms (I only encountered them back in the days when you ran a version of acrobat on Linux that did not yet support them), I could use a more elaborate description to understand how to use this module.